requirements/source/requirements-base.in

```diff
@@ -9,7 +9,7 @@ django-model-utils==2.2
 psycopg2==2.7.5
 sentry-sdk==0.19.3
 
-djangorestframework==3.1.3
+djangorestframework==3.2.5
```

Comment:
For my own understanding, if this requirement is in base why do we need it in the other requirements files? Does it not work otherwise 🤔